src/main.js

```diff
@@ -372,7 +372,10 @@ class ImageKitUppyPlugin extends Plugin {
             }
             formData.append("publicKey", this.opts.publicKey);
             formData.append("file", file.data);
-
+            if(this.opts.createdBy)
```

Comment:
Should check for presence of `createdBy.userId` if we are following this spec:
```
createdBy: { 
  userId: "user-id" 
}
```

Reply:
if the 
length is not correct then what needs to be done?